tests/python/frontend/keras/test_forward.py

```diff
@@ -443,6 +443,11 @@ def test_forward_crop(self, keras_mod):
         keras_model = keras_mod.models.Model(data, x)
         verify_keras_frontend(keras_model)
 
+        x = keras_mod.layers.Cropping2D(cropping=(2, 1))(data)
+        x = keras_mod.layers.Cropping2D(cropping=(1, 2))(x)
+        keras_model = keras_mod.models.Model(data, x)
+        verify_keras_frontend(keras_model, layout="NHWC")
```

Comment:
Could you also please add a test for `NCHW` layout?

Reply:
Thanks for your reminder, related test cases have been added.